### PR TITLE
add fetch request to Meetup API

### PR DESCRIPTION
**Pull request by xmd404**
@running-on-sunshine + @achoi2 

Please review. We can now make fetch requests to meetup API!!! ⚡️

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -21,14 +21,28 @@ passport.deserializeUser(function(user, done) {
...
 app.get('/', (req, res) => {
     res.render('index')
 });
 
+// fetch user profile from Meetup API
+fetch(
+    'https://api.meetup.com/members/self', 
+    {headers: new Headers(
+        {"Authorization": "Bearer b81eb53e6febf5a1cdf3487314faffae"}
+    )}
```
Comment (running-on-sunshine) [inline app.js:38]: This looks great and it works! 🙂 Awesome! Nice job, @theWebsiteKid! 

A thought: 
- In the headers section, 'Bearer' + alphanumeric value is supposed to change based on the user. As it currently stands, that special value in the script is currently tied to you.  I tried doing a fetch request while logging in and kept getting your data. 😄

File: package.json [hunk 2]
```diff
@@ -18,7 +18,9 @@
   "homepage": "https://github.com/achoi2/Buddy-up#readme",
   "dependencies": {
     "ejs": "^2.6.1",
+    "es6-promise": "^4.2.4",
     "express": "^4.16.3",
```
Comment (running-on-sunshine) [inline package.json]: Yay! More modules to play with! Cool! 🙂

It doesn't seem like we're currently using the 'es6-promise module' anywhere in app.js. Do you think we'll need it? If no, maybe we could do `npm uninstall es6-promise` on it.

Comment (xmd404) [reply package.json]: removed es6-promise

File: app.js (hunk 1 above)
Comment (xmd404) [reply app.js:38]: leave as is for now. this way we can work with a user object. we'll figure out how to make it dynamic later 👍

**Review by achoi2 — APPROVED** [review]
👍

File: app.js (hunk 1 above)
Comment (running-on-sunshine) [reply app.js:38]: Sounds good to me! 🙂

**Comment by running-on-sunshine** [comment]
This looks good! Great work @theWebsiteKid! 🙂

**Pull request merged**